.github/actions/lambda_deploy/action.yaml

```diff
@@ -24,11 +24,24 @@ runs:
       with:
         poetry-dir: py_gtfs_rt_ingestion
 
+    - name: Download psycopg2 with libpq installed
+      uses: actions/checkout@v2
+      with:
+        # this repo has a bunch of stars and bundles psycopg with libpq
+        repository: jkehler/awslambda-psycopg2
+        path: psycopg_with_libpq
+
     - name: Create build folder
       working-directory: py_gtfs_rt_ingestion
       run: |
+        # build the ingestion modules and install its dependencies to package
         poetry build
         poetry run pip install --upgrade -t package dist/*.whl
+        # remove the shipped version of psycopg2 and replace it with the one
+        # with libpq installed
+        rm -r package/psycopg2
+        mv ../psycopg_with_libpq/psycopg2-3.9 package/psycopg2
```

Comment:
this is the big change. use this version thats bundled with libpq instead of the one we install with pip.

Reply:
we're probably wasting some time installing psycopg2 with pip here, but i didn't want to waste more time mucking around with the .whl file to remove it.